src/DotNet.Status.Web/TimelineIssueTriage.cs

```diff
@@ -109,41 +108,46 @@ public async Task ProcessIssueEvent(IssuesHookData issuePayload)
 
                 // No duplicates, add label and move issue to triage
                 var issue = await gitHubClient.Issue.Get(issuePayload.Repository.Id, issuePayload.Issue.Number);
-                if (!issue.Labels.Any(l => l.Name == _labelName))
+                if (!issue.Labels.Any(l => l.Name == _markingLabelName))
                 {
                     var update = issue.ToUpdate();
-                    update.AddLabel(_labelName);
+                    update.AddLabel(_markingLabelName);
+                    foreach(var label in _issueLabels)
+                    {
+                        update.AddLabel(label);
```

Comment:
Question: would it be a problem if it already had for example "First Responder" and you add it again? Or the API handles duplicates just fine?

Reply:
It should not happen because it is gated by markingLabelName, but adding defensive code there could not harm.
I will add it